accounts-tequila-server.js

```diff
@@ -141,15 +141,18 @@ export function start (opts) {
 
   // Meteor login handlers (still) cannot be async functions:
   Accounts.registerLoginHandler((params) => Promise.await(tequilaLogin(
-    startOptions, protocol, params.tequilaKey)))
+    startOptions, protocol, params)))
 }
 
-async function tequilaLogin (opts, protocol, key) {
-  if (! key) { return undefined }
+async function tequilaLogin (opts, protocol, params) {
+  const key = params.tequilaKey
+  const auth_check = params.authCheckKey
 
-  debug("tequila.authenticate with key=" + key)
+  if (! key || ! auth_check) { return undefined }
```

Comment:
#personaltaste Please De Morgan this:

```
if (! (key && auth_check)) { ... }
```

Reply:
[Done](https://github.com/epfl-si/accounts-tequila/commit/5848d038545f30886ee2042a6ea0c0c81486bcbd)